readme.txt

```diff
@@ -4,7 +4,7 @@ Tags: job manager, job listing, job board, job management, job lists, job list,
 Requires at least: 5.5
 Tested up to: 5.7
```

Comment:
Are we good with bumping this to 5.8?

Reply:
Good catch! Updated here: https://github.com/Automattic/WP-Job-Manager/pull/2175/commits/5f30812b27c29dcb8a68441a67fd3f70537a04c2

Yep, I tested it and it's working well.